app/src/main/jni/audiolat.cpp

```diff
@@ -30,7 +30,8 @@ aaudio_data_callback_result_t dataCallback(AAudioStream *stream, void *userData,
                                            void *audioData,
                                            int32_t num_frames) {
   static int written_frames = 0;
-  static int buffer_index = 0;
+  static int play_buffer_index = 0;
```

Comment:
for coherence, can you make sure all the playout-related variables start with "playout_", and that the recorder ones start with "recorder_" ?

Reply:
Sorry. missed that name change.